winpkg/ddev.nsi

```diff
@@ -659,7 +616,8 @@ SectionEnd
  */
 LangString DESC_SecDDEV ${LANG_ENGLISH} "Install ${PRODUCT_NAME_FULL} (required)"
 LangString DESC_SecAddToPath ${LANG_ENGLISH} "Add the ${PRODUCT_NAME} (and sudo) directory to the global PATH"
-LangString DESC_SecMkcert ${LANG_ENGLISH} "mkcert (github.com/ FiloSottile/mkcert) is a simple tool for making locally-trusted development certificates. It requires no configuration"
+LangString DESC_SecSudo ${LANG_ENGLISH} "Sudo for Windows (github.com/mattn/sudo) allows for elevated privileges which are used to add hostnames to the Windows hosts file (required)"
```

Comment:
We haven't been using mattn/sudo for ages. We use gsudo.

Reply:
Done